src/pages/Facility/FacilitiesPage.tsx

```diff
@@ -71,8 +71,11 @@ export function FacilitiesPage() {
           selected={qParams.organization}
           onChange={(filter) => {
             if ("organization" in filter) {
-              if (filter.organization) {
-                setSelectedOrg(filter.organization as string);
+              if (
+                filter.organization &&
+                typeof filter.organization === "string"
```

Comment:
I think we should make sure that filter.organization is a string, instead of checking using typeof

Reply:
Ideally filter.organization should be string | undefined, and you can simply assign filter.organization